modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ErlangClientCodegen.java

```diff
@@ -117,6 +117,24 @@ public String getSwaggerType(Property p) {
         return type;
     }
 
+    public static String underscore2(String word) {
```

Comment:
@tsloughter Is there any issue with `underscore`? If that's the case, I would suggest fixing `underscore` instead of creating `underscore2`.

Reply:
Ah yea, bad name there. It isn't an issue that can be fixed in the original. The problem is `underscore` replaces `.` with `/`. Makes sense for creating paths in languages that utilize directories but for Erlang with its flat src directory and namespace we need to replace `.` with `_` as well.

Reply:
@tsloughter What about using `undercore(str.replaceAll(".", "_"))` instead?

Reply:
Ah, good idea, fixing.